manifests/03_credentials_request_alibaba.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: alibaba-disk-csi-driver-operator
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AlibabaCloudProviderSpec
+    statementEntries:
+    - action:
+      - ecs:AttachDisk
+      - ecs:DetachDisk
+      - ecs:DescribeDisks
+      - ecs:CreateDisk
+      - ecs:ResizeDisk
+      - ecs:CreateSnapshot
+      - ecs:DeleteSnapshot
+      - ecs:CreateAutoSnapshotPolicy
+      - ecs:ApplyAutoSnapshotPolicy
+      - ecs:CancelAutoSnapshotPolicy
+      - ecs:DeleteAutoSnapshotPolicy
+      - ecs:DescribeAutoSnapshotPolicyEX
+      - ecs:ModifyAutoSnapshotPolicyEx
+      - ecs:AddTags
+      - ecs:DescribeTags
+      - ecs:DescribeSnapshots
+      - ecs:ListTagResources
+      - ecs:TagResources
+      - ecs:UntagResources
+      - ecs:ModifyDiskSpec
+      - ecs:CreateSnapshot
+      - ecs:DescribeSnapshotGroups
+      - ecs:CreateSnapshotGroup
+      - ecs:DeleteSnapshotGroup
+      - ecs:CopySnapshot
+      - ecs:DeleteDisk
+      - ecs:DescribeInstanceAttribute
+      - ecs:DescribeInstances
+      - cs:CreateResourcesSystemTags
```

Comment:
Should this be `ecs:`? I couldn't find this action in the official docs.

Reply:
I asked Aliababa and they said `cs:` is correct